Tests/Functional/RunCommandTest.php

```diff
@@ -75,19 +75,6 @@ public function testRetry()
         $this->assertEquals(0, $job->getRetryJobs()->get(1)->getExitCode());
     }
 
-    public function testJobIsTerminatedIfMaxRuntimeIsExceeded()
```

Comment:
This feature does not support multiple nodes. So we disabled the feature already. Just remove the test, in case more people waste their time on it.